core/src/main/java/io/grpc/ClientCall.java

```diff
@@ -44,9 +44,9 @@
  * reasons for doing so would be the need to interact with flow-control or when acting as a generic
  * proxy for arbitrary operations.
  *
- * <p>{@link #start start()} must be called prior to calling any other methods. {@link #cancel} must
- * not be followed by any other methods, but can be called more than once, while only the first one
- * has effect.
+ * <p>{@link #start} or {@link #cancel} must be called prior
```

Comment:
It's a bit misleading to say you have to call cancel before other methods. How about:
> start must be called prior to calling any other methods, with the exception of cancel.

Or alternatively:
> start or cancel must be the first method called.